pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
+	// TODO: `/healthz` has been deprecated by `/`; keep it temporarily for
+	// backwards compatibility
+	healthzRoute               = "/healthz"
```

Comment:
Well. The problem is with customized images. We can't really depend on image tags. Hardcoding source registries of images isn't pretty either. We could move the decision to probe themselves - with an [`ExecAction`](http://kubernetes.io/v1.0/docs/user-guide/pod-states.html#container-probes). First there would be curl to `/` and then fallback to `/healthz`. Succesful endpoint could be cached on some tmpfs inside a container and would be used on subsequent checks. But I that's a bit of waste of resources.

No better solution come to my mind.